server/methods/core/cart.js

```diff
@@ -551,6 +562,36 @@ Meteor.methods({
       order.shipping = [];
     }
 
+    // Add current exchange rate into order.billing.currency
+    // If user currenct === shop currency, exchange rate = 1.0
+    const currentUser = Meteor.user();
+    let userCurrency = Reaction.getShopCurrency();
+    let exchangeRate = "1.00";
+
+    if (currentUser && currentUser.profile && currentUser.profile.currency) {
+      userCurrency = Meteor.user().profile.currency;
+    }
+
+    if (userCurrency !== Reaction.getShopCurrency()) {
+      const getExchangeRate = Meteor.call("shop/getCurrencyRates", userCurrency);
```

Comment:
I think normally we would only want to use names that start with `get` or `set` for function, not a variable.